fv3core/stencils/fv_dynamics.py

```diff
@@ -258,116 +250,142 @@ class DynamicalCore:
     def __init__(
         self,
         comm: fv3gfs.util.CubedSphereCommunicator,
-        namelist,
+        grid_data: GridData,
+        grid_indexing: GridIndexing,
+        damping_coefficients: DampingCoefficients,
+        config: DynamicalCoreConfig,
         ak: fv3gfs.util.Quantity,
         bk: fv3gfs.util.Quantity,
         phis: fv3gfs.util.Quantity,
     ):
         """
         Args:
             comm: object for cubed sphere inter-process communication
-            namelist: flattened Fortran namelist
+            grid_data: metric terms defining the model grid
+            grid_indexing: indexing information needed for stencils
+            damping_coefficients: damping configuration/constants
+            config: configuration of dynamical core
```

Comment:
maybe at least mention that this is often the namelist file settings here?

Reply:
I'll add a "for example" clause.

Reply:
Added.